hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -80,6 +80,10 @@ public Operation() {
         setFlag(true, BITMASK_CALL_TIMEOUT_64_BIT);
     }
 
+    public boolean executedLocally() {
```

Comment:
No. A backup is always executed on a different node then were it originated (primary). So it is always a remote call; and that is exactly what this method will return since callerAddress (primary) != this address.